helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/values.yaml

```diff
@@ -58,6 +58,7 @@ k8sMetadata:
     - release
   watch: true
   cache_ttl: 3600
+  propagate_namespace_labels: false
```

Comment:
Can you please add a comment to describe what it does?

Reply:
@harshit-splunk The comment has been added.